app/javascript/dashboard/routes/dashboard/search/components/SearchResultMessageItem.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <router-link :to="navigateTo" class="list-item">
+    <thumbnail :src="getThumbnail" size="42px" :username="getName" />
+    <div class="message-details">
+      <p class="name">{{ getName }}</p>
+      <div v-dompurify-html="messageContent" class="message-content" />
+    </div>
+  </router-link>
+</template>
+
+<script>
+import Thumbnail from 'dashboard/components/widgets/Thumbnail.vue';
+import messageFormatterMixin from 'shared/mixins/messageFormatterMixin';
+import { mapGetters } from 'vuex';
+import { frontendURL } from 'dashboard/helper/URLHelper.js';
+export default {
+  components: {
+    Thumbnail,
+  },
+  mixins: [messageFormatterMixin],
+  props: {
+    message: {
+      type: Object,
+      default: () => ({}),
+    },
+  },
+  computed: {
+    messageContent() {
+      return this.formatMessage(this.message.content);
+    },
+    ...mapGetters({
+      accountId: 'getCurrentAccountId',
+    }),
+    navigateTo() {
+      return frontendURL(
+        `accounts/${this.accountId}/conversations/${this.message.conversation_id}`
+      );
+    },
+    getThumbnail() {
```

Comment:
Adding these checks because, the api is not sending sender data for some message items in the payload.
CC: @tejaswinichile 
![image](https://user-images.githubusercontent.com/15716057/207407759-bc0d8f7d-2af7-41f5-9c3f-ba23cfa50fa9.jpg)